consoles/sshSerial.pm

```diff
@@ -0,0 +1,84 @@
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+#
+# Simple serial terminal over SSH
+
+package consoles::sshSerial;
+
+use strict;
+use warnings;
+
+use base 'consoles::console';
+
+use consoles::ssh_screen;
+
+use constant SSH_PTY_WIDTH => 2048;
```

Comment:
Maybe @okurz had more something like
```suggestion
use constant SSH_PTY_WIDTH => $ENV{OS_AUTOINST_SSH_PTY_WIDTH} // 2048;
```
in mind?

Reply:
Using `%ENV` to configure this is probably a bad idea. If anything, I could make it configurable through `add_console()` extra params. But otherwise most serial consoles already use PTY width hardcoded to 2048 cols:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/97fb6cf314f6e9c337b98056265eff460d3c36e6/lib/serial_terminal.pm#L124